src/components/add-to-a-list/template.test.js

```diff
@@ -23,6 +23,15 @@ const getTableItems = ($) => {
 };
 
 describe('Add to a list', () => {
+  it('should be rendered using a structure accessible for JAWS users', () => {
+    // See Adam Liptrot's comment on
```

Comment:
I would probably omit individuals' names as it's clear from the linked issue anyway.

Reply:
Good call, I'll update it!